Python/ast.c

```diff
@@ -958,7 +958,7 @@ PyAST_FromNode(const node *n, PyCompilerFlags *flags, const char *filename_str,
 
 }
 
-/* Return the AST repr. of the operator represented as syntax (|, ^, etc.)
+/* Return the AST repr of the operator represented as syntax (|, ^, etc.)
```

Comment:
Pretty sure "repr." here is short for "representation". This function has nothing to do with Python's `repr()`/`__repr__()`...

Reply:
- Agreed, repr expanded to representation
- Extra line between function header and function body also deleted